### PR TITLE
Minor: code cleanup in _process()

### PR DESCRIPTION
**Pull request by hroff-1902**
* Pythonic stuff moved to separate methods in order to make _process() more straight so that the sequence of actions in _process() will be more readable.
* Fixed typo in a comment

### CODE REVIEW

**Comment by hroff-1902** [comment]
Comments adjusted...
Almost perfect now :-)

File: freqtrade/freqtradebot.py [hunk 1]
```diff
@@ -204,6 +200,24 @@ def _process(self) -> bool:
             self.state = State.STOPPED
         return state_changed
 
+    def _query_trades(self) -> List[Any]:
```
Comment (xmatthias) [inline freqtrade/freqtradebot.py]: nice cleanup!
I personally would move this to `persistence.py` and call it `Trade.get_open()` (or similar)...

also - `Trade.query.filter(Trade.is_open.is_(True)).all()` appears 3 times in this file alone, and another 3 times in `rpc.py` - if we move it to a function, we should replace all of them ...

Comment (hroff-1902) [reply freqtrade/freqtradebot.py]: Actually I guess the whole Freqtradebot class should be split into 2 classes in the future in order to split the app start/stop/loop/reconfig logic and trading business logic which starts in the _process...

Comment (hroff-1902) [reply freqtrade/freqtradebot.py]: > nice cleanup!
> I personally would move this to `persistence.py` and call it `Trade.get_open()` (or similar)...
> 
> also - `Trade.query.filter(Trade.is_open.is_(True)).all()` appears 3 times in this file alone, and another 3 times in `rpc.py` - if we move it to a function in persistence, we could replace all of them at once...

Fine... :+1:

Comment (xmatthias) [reply freqtrade/freqtradebot.py]: Absolutely, even more pieces if you will ... has been discussed in #1335 if you're looking for more about that

**Comment by mishaker** [comment]
Nice job ! defo take a look at #1335 too :)

**Comment by hroff-1902** [comment]
> take a look at #1335 too :)

Already have taken, reading it diagonally forawhile. It's too long :-)

**Pull request merged**